weave

```diff
@@ -1231,10 +1200,10 @@ launch() {
     fi
 
     # Create a data-only container for persistence data
-    if ! docker inspect -f ' ' $DB_CONTAINER_NAME > /dev/null 2>&1 ; then
-       protect_against_docker_hang
-       docker create -v /weavedb --name=$DB_CONTAINER_NAME \
-           --label=weavevolumes $WEAVEDB_IMAGE >/dev/null
+    if ! util_op container-state $DB_CONTAINER_NAME > /dev/null 2>&1 ; then
```

Comment:
Why `container-state` rather than `container-id`, say ?  We don't expect this one to be restarting, etc.

Reply:
Yeah, I just liked the idea of using a command named `container-state` rather than `container-id` to fetch the state of a container.  It read like self-documenting code to me.  Maybe there should be command `is-container-running` or something like that, which is an alias for `container-id`?

Reply:
ok, `container-state` is fine.